subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -724,8 +724,8 @@ public void setBuildCacheDebugLogging(boolean buildCacheDebugLogging) {
      * @since 6.5
      */
     @Incubating
-    public void setWatchFileSystem(boolean watchFileSystem) {
-        this.watchFileSystem = watchFileSystem;
+    public boolean isWatchFileSystem() {
+        return watchFileSystem;
     }
```

Comment:
Looking at `isBuildCacheEnabled()` I think this should be called `isWatchFileSystemEnabled()`.

Reply:
Hmm... There is also `isConfigureOnDemand()`, `isBuildScan()`, `isContinuous()`, etc. So it seems like `isBuildCacheEnabled()` is the odd one.

I would keep `isWatchFileSystem()`. WDYT?

Reply:
I don't feel very strongly, but `isWatchFileSystemEnabled()` sounds slightly more readable to me. Up to you.